ApplyToTheIndustry/Assets/Scripts/ResumeBuilder/Resume.cs

```diff
@@ -15,7 +15,9 @@ public class Resume : MonoBehaviour
     public TextMeshProUGUI resumeTMPSkillList;
     public string resumeMessageDefault;
 
-
+    // Onboarding fields
+    public int neededClicks = 3;
+    public int tutorialClicks = 0;
```

Comment:
Some other systems (i.e. booking courses for instance) might require tutorials as well. Seems like this could be replaced by another component, a sort of TutorialConfig component class, that classes such as these can reference or have an instance of.